tron/utils/scribereader.py

```diff
@@ -183,6 +186,15 @@ def read_log_stream_for_action_run(
         finally:
             stream.close()
 
+    if truncated_output:
+        formated_today = datetime.datetime.now(datetime.timezone.utc).strftime("%Y-%m-%dT%H:%M:%S.%fZ")
```

Comment:
we could probably skip having to create a datetime object if we just add this into the very end of the list that we return (in the same way that we add in the malformed lines output)

Reply:
we sort the list by timestamp before returning it because we're occasionally getting data out of order from scribereader

Reply:
correct - but we know we want this at the very end, so we can just insert at the end